src/app/features/de-duplication/bulk-merge-records/bulk-merge-records.component.ts

```diff
@@ -117,53 +112,12 @@ export class BulkMergeRecordsComponent<E extends Entity> implements OnInit {
       }
     });
   }
-  private setInitialMergedValues(): void {
+  private setFieldDisabledProperty(): void {
     for (const field of this.fieldsToMerge) {
-      const valueA = this.entitiesToMerge[0][field.id];
-      const valueB = this.entitiesToMerge[1][field.id];
       this.isFieldDisabled[field.id] = this.entitiesToMerge.map(
         (entity) => !this.hasValue(entity[field.id]),
       );
-      const control = this.mergeForm.formGroup.get(field.id);
-      if (!control) continue;
-
-      const mergedValue = this.setSmartSelectedValue(
-        valueA,
-        valueB,
-        control.value,
-      );
-      control.setValue(mergedValue);
-    }
-  }
-
-  /**
-   * Determines the best value to use when merging two entity field values.
-   * If both values are identical, it returns one of them.
-   * If one value is empty while the other is not, it returns the non-empty value.
-   * Otherwise, it retains the current form value.
-   */
-  private setSmartSelectedValue(
-    valueA: any,
-    valueB: any,
-    currentValue: any,
-  ): any {
-    if (this.areValuesIdentical(valueA, valueB)) {
-      return valueA;
-    } else if (!this.hasValue(valueA) && this.hasValue(valueB)) {
-      return valueB;
-    } else if (!this.hasValue(valueB) && this.hasValue(valueA)) {
-      return valueA;
-    } else if (Array.isArray(valueA) || Array.isArray(valueB)) {
-      return [
-        ...(Array.isArray(valueA) ? valueA : []),
-        ...(Array.isArray(valueB) ? valueB : []),
-      ];
     }
-    return currentValue;
-  }
-
-  private areValuesIdentical(a: any, b: any): boolean {
-    return JSON.stringify(a) === JSON.stringify(b);
   }
 
   private allowsMultiValueMerge(field?: FormFieldConfig): boolean {
```

Comment:
This can probably also be moved into the merge-fields component, as it only relates to a specific field?